lib/react_on_rails/helper.rb

```diff
@@ -328,10 +324,14 @@ def load_pack_for_component(component_name)
 
       ReactOnRails::WebpackerUtils.raise_nested_entries_disabled unless ReactOnRails::WebpackerUtils.nested_entries?
 
-      append_javascript_pack_tag "generated/#{component_name}"
-      append_stylesheet_pack_tag "generated/#{component_name}"
+      ApplicationController.helpers.append_javascript_pack_tag("generated/#{component_name}")
+      ApplicationController.helpers.append_stylesheet_pack_tag("generated/#{component_name}")
```

Comment:
My concern is that this change is ONLY related to running view helper specs:

```
Failure/Error: cached_react_component("App", cache_key: "cache-key", &props)

NameError:
  undefined local variable or method `react_component_name' for #<RSpec::ExampleGroups::ReactOnRailsProHelper::CachedReactComponent::Caching::ReactOnRailsProHelperCachedReactComponent:0x0000563a9d6279c8>
  Did you mean?  react_component
                 react_component_hash
/home/circleci/project/app/helpers/react_on_rails_pro_helper.rb:20:in `fetch_react_component'
/home/circleci/project/app/helpers/react_on_rails_pro_helper.rb:48:in `block in cached_react_component'
/home/circleci/project/lib/react_on_rails_pro/utils.rb:107:in `with_trace'
/home/circleci/project/app/helpers/react_on_rails_pro_helper.rb:45:in `cached_react_component'
./spec/helpers/react_on_rails_pro_helper_spec.rb:84:in `block (6 levels) in <top (required)>'
./spec/helpers/react_on_rails_pro_helper_spec.rb:83:in `block (5 levels) in <top (required)>'
/home/circleci/project/spec/react_on_rails_pro/support/caching.rb:26:in `block (2 levels) in <main>'
```

and unrelated to real usage of the view helper.

We should never change the code just to help the tests.

Reply:
Does this code work?

@Judahmeek I found that loading pack tag through `ApplicationController.helpers` brokes auto-load functionality (it doesn't load packs).

Is it possible to `include` required modules into `ReactOnRailsProHelper`?

CC: @pulkitkkr @justin808

Reply:
@frullah Can you name the branch you used to test this?

Reply:
@Judahmeek try this repo: https://github.com/frullah/react_on_rails_auto_load_error

- branch that work: loading-packs-not-through-application-controller-helpers.
- branch that doesn’t work: loading-packs-not-through-application-controller-helpers.

1_. run setup
bundle
yarn

2_. run app
bin/webpacker
rails s

3_. access http://localhost:3000/hello_world, and see if the packs is loaded on the browser or not.